core/src/test/resources/test_assets.json

```diff
@@ -22,6 +22,53 @@
         "min_amount": 1,
         "max_amount": 1000000
       },
+      "sep6": {
+        "deposit": {
```

Comment:
For withdraw and deposit, we do not use SEP specific configurations. This may results in inconsistency with SEP24 configuration.

Reply:
Yes, but these fields are specific to SEP-6. `fields` can be moved into the root `deposit` and `types` can be moved into the root `withdraw`, however, they are only relevant if `sep6_enabled: true`. 

I'm fine with this approach, but what about `exchange_enabled`? That is only relevant to SEP-6 at the moment.

Reply:
>  but what about `exchange_enabled`? That is only relevant to SEP-6 at the moment.

If we follow Jake's suggestion below and always assume `exchange_enabled` is true, then this is not a problem.